tests/test_frame.py

```diff
@@ -2071,17 +2139,23 @@ def test_types_regressions() -> None:
     s1 = pd.Series([1, 2, 3])
     s2 = pd.Series([4, 5, 6])
     df = pd.concat([s1, s2], axis=1)
-    ss1: pd.Series = pd.concat([s1, s2], axis=0)
-    ss2: pd.Series = pd.concat([s1, s2])
+    # TODO the inference here returns Any, should return Series
+    ts1: pd.Series = pd.concat([s1, s2], axis=0)
+    ts2: pd.Series = pd.concat([s1, s2])
 
     # https://github.com/microsoft/python-type-stubs/issues/110
+    # TODO the type inference below is broken, a Timestamp is not inferring as datetime.date
     d: datetime.date = pd.Timestamp("2021-01-01")
```

Comment:
Use this:
```python
    check(assert_type(pd.Timestamp("2021-01-01"), pd.Timestamp), datetime.date)
```
You can't use `assert_type()` to assert that an object is a subclass of a type.